test/remote/account_signin_verification_enable_tests.js

```diff
@@ -9,8 +9,8 @@ var Client = require('../client')
 test(
   'signin confirmation can be disabled',
   function (t) {
-    process.env.SIGNIN_CONFIRMATION_ENABLED = false
     var config = require('../../config').getProperties()
+    process.env.SIGNIN_CONFIRMATION_ENABLED = false
```

Comment:
I had to swap this around because the test started failing. There was a change to `.env.dev` file that enabled sign-in confirmation by default, which changed how convict was setting values.